devguide/howtos/add-function.md

```diff
@@ -63,8 +63,19 @@ The parameters passed to call are all optional (except ~name and ~in_product_sin
   permitted to merely be a single value rather than a list of values. If no
   ~result is specified, the default is unit.
 
+- The ~doc parameter describes what it is, the message is doing.
+
+- The ~hide_from_docs parameter prevents te message from being included in the document ation when generated. This is indicated by a bool.
+
+- The ~pool_internal parameter is used to indicate if the message should be callable by all or only internal hosts. This is indicated by a bool. 
```

Comment:
It'd be shorter to say  'pool_internal bool parameter'

Reply:
fixed